evcouplings/visualize/pairs.py

```diff
@@ -1183,43 +1182,101 @@ def enrichment_pymol_script(enrichment_table, output_file,
         "cartoon putty"
     chain : str, optional (default: None)
         Use this PDB chain in residue selection
+    legacy: bool, optional (default: False)
+        Use legacy (2011) red and yellow colormap
+        for EC enrichment
     """
-    t = enrichment_table.query("enrichment > 1")
+    if legacy: 
+        t = enrichment_table.query("enrichment > 1")
+        t.loc[:, "b_factor"] = t.enrichment
+        # compute boundaries for highly coupled residues
+        # that will be specially highlighted
+        boundary1 = int(0.05 * len(t))  # top 5%
+        boundary2 = int(0.15 * len(t))  # top 15%
+
+        t.loc[:, "b_factor"] = t.enrichment
+
+        # set color for "low" enrichment (anything > 1)
+        t.loc[:, "color"] = "yelloworange"
 
-    # compute boundaries for highly coupled residues
-    # that will be specially highlighted
-    boundary1 = int(0.05 * len(t))  # top 5%
-    boundary2 = int(0.15 * len(t))  # top 15%
+        # high
+        t.loc[t.iloc[0:boundary1].index, "color"] = "red"
 
-    t.loc[:, "b_factor"] = t.enrichment
+        # medium
+        t.loc[t.iloc[boundary1:boundary2].index, "color"] = "orange"
+        
+    else:
+        t = deepcopy(enrichment_table)
```

Comment:
Not that it really matters here, at some point I learnt that there is DataFrame.copy() method which saves one from using the deepcopy package

Reply:
good to know - I left as is because deepcopy is used other places in this file, so we wouldn't be removing a dependency